modules/machineset-label-gpu-autoscaler.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * machine_management/applying-autoscaling.adoc
+// * machine_management/creating_machinesets/creating-machineset-aws.adoc
+// * machine_management/creating_machinesets/creating-machineset-azure.adoc
+// * machine_management/creating_machinesets/creating-machineset-azure-stack-hub.adoc
+// * machine_management/creating_machinesets/creating-machineset-bare-metal.adoc
+// * machine_management/creating_machinesets/creating-machineset-gcp.adoc
+// * machine_management/creating_machinesets/creating-machineset-ibm-cloud.adoc
+// * machine_management/creating_machinesets/creating-machineset-ibm-power-vs.adoc
+// * machine_management/creating_machinesets/creating-machineset-nutanix.adoc
+// * machine_management/creating_machinesets/creating-machineset-osp.adoc
+// * machine_management/creating_machinesets/creating-machineset-vsphere.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="machineset-label-gpu-autoscaler_{context}"]
+= Labeling GPU machine sets for the cluster autoscaler
+
+You can use a machine set label to indicate which machines the cluster autoscaler can use to deploy GPU-enabled nodes.
+
+.Prerequisites
+* Your cluster uses a cluster autoscaler.
+
+.Procedure
+
+* Add a `cluster-api/accelerator` label to the machine set that you want to create machines for the cluster autoscaler to use to deploy GPU-enabled nodes:
```

Comment:
```suggestion
* On the machine set where you want to create machines for the cluster autoscaler to use to deploy GPU-enabled nodes, add a `cluster-api/accelerator` label:
```
This one is tricky! I think I understand your original text, but I had to read it a couple times. I'm honestly not sure if my revision is much better. I wanted to divide the line into 2 sentences, but I couldn't think of a way to do that without sounding awkward 😅

Reply:
I think it still needs to be "that" rather than "where" (it's the machine set _that_ will do XYZ, as opposed to some other machine set), but I like how you flipped this around. I honestly really struggle with how to best state these long restrictive clauses (and I have many such things in my content areas :sweat_smile:)